### PR TITLE
Copter: run the "point yaw to ROI" controller at full rate (400Hz) in…

### PR DESCRIPTION
**Pull request by amilcarlucas**
…stead of 1/4 of full rate (100Hz)

This should improve pointing at ROI and replaces #11172

Removes 14 lines of code

### CODE REVIEW

File: ArduCopter/autoyaw.cpp [hunk 1]
```diff
@@ -5,12 +5,7 @@ Mode::AutoYaw Mode::auto_yaw;
 // roi_yaw - returns heading towards location held in roi
 float Mode::AutoYaw::roi_yaw()
 {
-    roi_yaw_counter++;
-    if (roi_yaw_counter >= 4) {
-        roi_yaw_counter = 0;
-        _roi_yaw = get_bearing_cd(copter.inertial_nav.get_position(), roi);
-    }
-
+    _roi_yaw = get_bearing_cd(copter.inertial_nav.get_position(), roi);
```
Comment (peterbarker) [inline ArduCopter/autoyaw.cpp:8]: Why are we still storing this?  Can we eliminate the member variable?

Comment (khancyr) [reply ArduCopter/autoyaw.cpp:8]: yep the _roi_yaw could be remove

Comment (amilcarlucas) [reply ArduCopter/autoyaw.cpp]: done

**Comment by peterbarker** [comment]
... OK, so can the method now be const?

Sorry :-)

**Comment by amilcarlucas** [comment]
Done

**Review by peterbarker — CHANGES_REQUESTED** [review]
looks nicer with each iteration.

Squash into one commit, please.

**Review by peterbarker — APPROVED** [review]
LGTM

**Comment by peterbarker** [comment]
Labelled devcalltopic in case it doesn't get merged before then

**Pull request merged**